src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -528,7 +528,7 @@ DataCallbackResult AudioStreamAAudio::callOnAudioReady(AAudioStream * /*stream*/
         if (result == DataCallbackResult::Stop) {
             LOGD("Oboe callback returned DataCallbackResult::Stop");
         } else {
-            LOGE("Oboe callback returned unexpected value = %d", result);
+            LOGE("Oboe callback returned unexpected value. Error: %s", oboe::convertToText(result));
```

Comment:
In this case, we want to know what strange integer value was returned. So the convertToText() will probably not handle it. So better to cast to (int) and print with %d.